static/css/theme.css

```diff
@@ -988,4 +988,21 @@ pre .copy-to-clipboard:hover {
     transform-style: preserve-3d;
 }
 
+/* Add checkmark icon align to right and color*/
+#sidebar ul.topics > li > a .fa {
+    margin-top: 9px;
+}
+#sidebar ul li .fa {
```

Comment:
@Xipas Did not see it at the first time, but if we define an font awesome icon in the prefix of a menu item, it's not shown. We should add a class for the check icon, in order to be specific.